loadgen/loadgen.cc

```diff
@@ -831,7 +833,7 @@ void PerformanceSummary::Log(AsyncSummary& summary) {
 
   switch (settings.scenario) {
     case TestScenario::SingleStream: {
-      summary("90th percentile latency (ns) : ", latency_target.value);
+      summary("90th percentile latency (ns) : ", latency_percentiles[1].value);
```

Comment:
Hmm. We probably need a single_stream_target_latency_percentile too.

Reply:
I think so.